src/DataFactory/DataFactoryV2/help/Set-AzDataFactoryV2.md

```diff
@@ -74,12 +201,57 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -HostName
+The tags of the data factory.
```

Comment:
@sanjkuma this description needs to be updated

Reply:
Done